plugins/redirect/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "gatsby-plugin-meta-redirect",
```

Comment:
We should mark this as "private", and remove the "repository" otherwise, it could confuse NPM.

Reply:
There's nothing for it to be confused about. No dependencies exist so there is nothing to install. It also works with an empty object. The only reason it's filled in is to provide credit for the original plugin author.

Reply:
I would rather still have `private`, if possible. In the eventuality we ever use monorepos... Well, this outa be a problem.